### PR TITLE
make scripts run from anywhere

### PR DESCRIPTION
**Pull request by YunziZhangAzure**
## Problem
<Please describe what problem this PR is trying to resolve>
saplibrary_tfstate_script.sh and saplibrary_tfstate_script.sh should be able to be run anywhere.

## Solution
<Please elaborate the solution for the problem>

## Tests
<Please provide steps to test the PR>

image

## Notes
<Additional comments for the PR>

### CODE REVIEW

File: deploy/terraform/bootstrap/sap_library/deployer_tfstate_script.tmpl [hunk 1]
```diff
@@ -24,23 +32,25 @@ function validate_deployer_remote_state(){
...
-    cp ${deployer_terraform_tfstate_path} ../
+    cd $${local_file_dir}/../
 
-    cd ..
+    cp ${deployer_terraform_tfstate_path} ./
 
-    echo "Clean .terraform if exists:"
+    printf "%s\n" "Clean local .terraform if exists:"
 
     rm -r .terraform
```
Comment (rukawata) [inline deploy/terraform/bootstrap/sap_library/deployer_tfstate_script.tmpl:50]: error handling can be improved here.
When there is no `.terraform` folder, we see error which is expected behavior if it is the first time running the script.
`rm: cannot remove '.terraform': No such file or directory`

File: deploy/terraform/bootstrap/sap_library/deployer_tfstate_script.tmpl [hunk 2]
```diff
@@ -24,23 +32,25 @@ function validate_deployer_remote_state(){
...
 
+    printf "%s\n" "Start terraform execution"
```
Comment (rukawata) [inline deploy/terraform/bootstrap/sap_library/deployer_tfstate_script.tmpl:52]: IMO, execution is when one runs terraform apply.
This is more backend config, or if we want to make it more vague, remote state preparation.
What you have a couple of lines above is more appropriate: 
`printf "%s\n" "Start initializing remote backend for deployer"`

Comment (YunziZhangAzure) [reply deploy/terraform/bootstrap/sap_library/deployer_tfstate_script.tmpl:52]: Will remove this

Comment (YunziZhangAzure) [reply deploy/terraform/bootstrap/sap_library/deployer_tfstate_script.tmpl:52]: Will remove it

File: deploy/terraform/bootstrap/sap_library/deployer_tfstate_script.tmpl (hunk 1 above)
Comment (YunziZhangAzure) [reply deploy/terraform/bootstrap/sap_library/deployer_tfstate_script.tmpl:50]: Will change it  to rm -rf

**Pull request merged**